src/main/java/net/snowflake/client/core/QueryContextEntryDTO.java

```diff
@@ -0,0 +1,52 @@
+package net.snowflake.client.core;
+
+public class QueryContextEntryDTO {
```

Comment:
/**
 * An entry in the set of query context exchanged with Cloud Services.
 * This includes a domain identifier(id), a timestamp that is monodically increasing, a priority for eviction and the opaque.
 */
 * information sent from the Cloud service.

Reply:
added this comment